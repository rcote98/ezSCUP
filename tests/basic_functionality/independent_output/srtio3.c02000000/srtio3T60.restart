2	2	2
5	3
Sr	Ti	O
3.00000000E-02	3.00000000E-02	0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	1	1	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	2	2	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	3	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	4	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	5	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	1	1	1	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	1	2	2	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	1	3	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	1	4	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	1	5	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	1	0	1	1	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	1	0	2	2	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	1	0	3	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	1	0	4	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	1	0	5	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	1	1	1	1	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	1	1	2	2	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	1	1	3	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	1	1	4	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	1	1	5	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	0	0	1	1	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	0	0	2	2	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	0	0	3	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	0	0	4	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	0	0	5	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	0	1	1	1	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	0	1	2	2	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	0	1	3	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	0	1	4	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	0	1	5	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	1	0	1	1	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	1	0	2	2	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	1	0	3	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	1	0	4	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	1	0	5	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	1	1	1	1	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	1	1	2	2	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	1	1	3	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	1	1	4	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
1	1	1	5	3	0.00000000E+00	0.00000000E+00	0.00000000E+00
