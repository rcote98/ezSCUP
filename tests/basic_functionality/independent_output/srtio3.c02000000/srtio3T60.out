
*****************************************************************
*                          Welcome to                           *
*                           SCALE-UP                            *
*                                                               *
*            Second-Principles Computational Approach           *
*                             for                               *
*                     Lattice and Electrons                     *
*                                                               *
*                       Version:         1.0.0.Gorbea           *
*                    Build date: 20200801T181                   *
*****************************************************************
*                    SCALE-UP is property of                    *
*              the University of Cantabria (Spain)              *
*    and the Luxembourg Institute of Science and Technology     *
*****************************************************************
* Authors:                                                      *
*                                                               *
*  - Pablo Garcia-Fernandez   garciapa@unican.es                *
*  - Jorge Iñiguez            jorge.iniguez@list.lu             *
*  - Javier Junquera          junqueraj@unican.es               *
*****************************************************************
* SCALE-UP is based on two earlier programs:                    *
*                                                               *
* -  SPLD ('Second-Principles Lattice Dynamics'), developed     *
*    at the ICMAB-CSIC (Spain) by J.C. Wojdel, M.P. Ljungberg   *
*    and J. Iñiguez, and distributed under MIT license.         *
*                                                               *
* -  MKII a second-principles electron code                     *
*    developed at the University of Cantabria by                *
*    P. Garcia-Fernandez  and J. Junquera.                      *
*****************************************************************
*                                                               *
* Bibliography:                                                 *
*                                                               *
*  P. Garcia-Fernandez et al. Phys. Rev. B 93, 195137 (2016)    *
*  J. C. Wojdel et al., J. Phys. Cond. Matt. 25, 305401 (2013)  *
*****************************************************************
Reading parameter file... lattice... done!
Reading restart file     srtio3T60.restart                                                               

Basic data:
  Number of CPUs:               1
  Run mode:            Montecarlo
  Lattice:                 Active
  Electrons:           NOT Active
  External electric field, static. Field components (V/m):   0.000000E+00   0.000000E+00   0.000000E+00

Reference system:
 Supercell data:
  Lattice constants (Ang):   7.690   7.690   7.690
  Lattice angles (degree):  90.000  90.000  90.000
 Supercell size:   2   2   2

 Total number of atoms          :      40
 Total number of orbitals       :       0
 Total number of active orbitals:       0

 Atom in unit cell:
 Atom   Label       Mass  Charge 
    1   Sr         87.60     0.0
    2   Ti         47.90     0.0
    3   O          16.00     0.0
    4   O          16.00     0.0
    5   O          16.00     0.0

 Orbitals in unit cell:
 Orbital   Label  

 Background dielectric tensor:
    6.2319     0.0000     0.0000
    0.0000     6.2319    -0.0000
    0.0000    -0.0000     6.2319

Initial values (basic output MC simulation)
Energy (eV):            -0.298472E+05
Polarization (C/m2):     0.000000    0.000000    0.000000
Strain:                  0.030000    0.030000    0.000000    0.000000    0.000000    0.000000

LT:    Iter       Temp(K)    Energy(eV)   Pol_x(C/m2)   Pol_y(C/m2)   Pol_z(C/m2) Av_Pl_x(C/m2) Av_Pl_y(C/m2) Av_Pl_z(C/m2)       Strn_xx       Strn_yy       Strn_zz       Strn_yz       Strn_xz       Strn_xy    Av_Strn_xx    Av_Strn_yy    Av_Strn_zz    Av_Strn_yz    Av_Strn_xz    Av_Strn_xy 
LT:      10  0.600000E+02 -0.298471E+05 -0.621191E-01  0.576480E-01  0.199584E-01 -0.397969E-01  0.323487E-01  0.310587E-01  0.300000E-01  0.300000E-01 -0.264806E-02  0.356620E-03  0.100742E-02  0.000000E+00  0.330000E-01  0.330000E-01 -0.153058E-02  0.408017E-03  0.759191E-03  0.000000E+00 
LT:      20  0.600000E+02 -0.298472E+05 -0.106216E+00  0.118616E+00  0.347540E-01 -0.614856E-01  0.746160E-01  0.349393E-01  0.300000E-01  0.300000E-01 -0.625473E-02 -0.116994E-02 -0.632947E-06  0.000000E+00  0.315000E-01  0.315000E-01 -0.291130E-02 -0.186968E-03  0.621138E-03  0.000000E+00 
LT:      30  0.600000E+02 -0.298472E+05 -0.147875E+00  0.150552E+00  0.768114E-02 -0.905782E-01  0.937566E-01  0.235303E-01  0.300000E-01  0.300000E-01 -0.766524E-02 -0.120344E-02  0.270483E-02  0.000000E+00  0.310000E-01  0.310000E-01 -0.443706E-02 -0.592573E-03  0.708396E-03  0.000000E+00 
LT:      40  0.600000E+02 -0.298471E+05 -0.911464E-01  0.229848E+00  0.312082E-01 -0.994427E-01  0.121620E+00  0.212547E-01  0.300000E-01  0.300000E-01 -0.681496E-02 -0.183157E-02  0.434033E-03  0.000000E+00  0.307500E-01  0.307500E-01 -0.510629E-02 -0.902961E-03  0.108052E-02  0.000000E+00 
LT:      50  0.600000E+02 -0.298472E+05 -0.122273E+00  0.216237E+00  0.240563E-01 -0.993066E-01  0.139687E+00  0.210947E-01  0.300000E-01  0.300000E-01 -0.821434E-02 -0.197190E-02  0.110192E-02  0.000000E+00  0.306000E-01  0.306000E-01 -0.544632E-02 -0.114652E-02  0.116200E-02  0.000000E+00 
LT:      60  0.600000E+02 -0.298473E+05 -0.192209E+00  0.266100E+00  0.428931E-03 -0.104867E+00  0.155273E+00  0.177571E-01  0.300000E-01  0.300000E-01 -0.127148E-01 -0.325463E-02 -0.168204E-03  0.000000E+00  0.305000E-01  0.305000E-01 -0.645474E-02 -0.139602E-02  0.111389E-02  0.000000E+00 
LT:      70  0.600000E+02 -0.298473E+05 -0.152307E+00  0.194297E+00  0.210851E-01 -0.115686E+00  0.167938E+00  0.154621E-01  0.300000E-01  0.300000E-01 -0.120134E-01 -0.851604E-02 -0.335730E-03  0.000000E+00  0.304286E-01  0.304286E-01 -0.722346E-02 -0.186791E-02  0.870487E-03  0.000000E+00 
LT:      80  0.600000E+02 -0.298472E+05 -0.204140E+00  0.276313E+00 -0.154700E-01 -0.122575E+00  0.175294E+00  0.912396E-02  0.300000E-01  0.300000E-01 -0.205685E-01 -0.846338E-02  0.408771E-02  0.000000E+00  0.303750E-01  0.303750E-01 -0.826545E-02 -0.274087E-02  0.848737E-03  0.000000E+00 
LT:      90  0.600000E+02 -0.298473E+05 -0.243828E+00  0.220991E+00  0.105122E-01 -0.134532E+00  0.180056E+00  0.765079E-02  0.300000E-01  0.300000E-01 -0.153279E-01 -0.700934E-02  0.613447E-03  0.000000E+00  0.303333E-01  0.303333E-01 -0.931915E-02 -0.328788E-02  0.103791E-02  0.000000E+00 
LT:     100  0.600000E+02 -0.298473E+05 -0.298452E+00  0.157320E+00  0.903588E-02 -0.145711E+00  0.182341E+00  0.577973E-02  0.300000E-01  0.300000E-01 -0.201547E-01 -0.486035E-02 -0.335580E-02  0.000000E+00  0.303000E-01  0.303000E-01 -0.102805E-01 -0.350661E-02  0.652684E-03  0.000000E+00 

Final values (basic output MC simulation)
Energy (eV):            -0.298473E+05
Polarization (C/m2):    -0.298452    0.157320    0.009036
Strain:                  0.030000    0.030000   -0.020155   -0.004860   -0.003356    0.000000

-----------------------------------------------------------------
 Backing up the input read by fdf:

System_name     srtio3T60
Print_routineinfo         5     # default value
Print_routinedepth         0     # default value
Print_inout         0     # default value
Print_interactions         0     # default value
Print_k-vectors         0     # default value
Print_energies         1     # default value
Print_wavefunctions        -1     # default value
Print_electrostatic         1     # default value
Print_orbocc         1     # default value
Print_bands        -1     # default value
Print_forces        -1     # default value
Print_detail        -1     # default value
Print_Just_Geometry        -1     # default value
Print_Velocity         0     # default value
Print_Space_Hamiltonian        -1     # default value
Print_space_denmat        -1     # default value
Print_Geometry         1     # default value
Print_geoformat     normal     # default value
Print_test_file        -1     # default value
Print_local_lattice_energy        -1     # default value
Print_local_current        -1     # default value
Print_std_electron_nsteps         1     # default value
Print_std_lattice_nsteps             10
Print_std_energy              T
Print_std_polarization              T
Print_std_current              F     # default value
Print_std_strain              T
Print_std_av_polarization              T
Print_std_av_strain              T
Print_std_av_energy              F     # default value
Print_std_delta_energy              F     # default value
Print_std_temperature              T     # default value
Print_std_maximum_force              F     # default value
Print_std_maximum_stress              F     # default value
Print_std_mdd              T     # default value
Print_std_fermi              T     # default value
Print_std_pulay              T     # default value
Print_std_neh              T     # default value
Print_std_el_pol              T     # default value
Print_std_micro              F     # default value
run_mode     monte_carlo
Print_std_mdd              F     # default value
Print_std_fermi              F     # default value
Print_std_pulay              F     # default value
Print_std_neh              F     # default value
Print_std_electron_nsteps     10000     # default value
No_electron
%block fix_strain_component
 T T F F F T
T                                                                                             T
T                                                                                             T
F                                                                                             F
F                                                                                             F
F                                                                                             F
T                                                                                             T
Parameter_file     srtio3_full_lat.xml
%block Supercell
 2 2 2
%endblock Supercell
electric_field_scale          1.000000000     v/m     # default value
%block static_electric_field
 0.0 0.0 0.0
temperature         0.1000000000E-98 kelvin     # default value
%block external_stress
 0.0 0.0 0.0 0.0 0.0 0.0
Charge_tolerance         0.1000000000E-07     # default value
Geometry_restart
Geometry_restart     srtio3T60.restart
MC_nsweeps
MC_nsweeps            100
MC_temperature
MC_temperature          60.00000000         kelvin
# above item originally: MC_temperature          60.00000000     kelvin
MC_annealing_rate          1.000000000
MC_init_step_d         0.9448629925E-01 bohr     # default value
MC_init_step_s         0.1889725985E-02 bohr     # default value
MC_max_step_d         0.9448634389           bohr
# above item originally: MC_max_step_d         0.5000000000     ang
MC_min_step_d         0.1889725985E-05 bohr     # default value
MC_target_acc_d         0.5000000000         # default value
MC_target_acc_s         0.5000000000         # default value
MC_strains              T
n_write_mc
n_write_mc             10
Print_std_lattice_nsteps
*****************************************************************
 Memory usage:
  Function             Total_memory(Mb)  Max_memory(Mb)      Node
  setup_main                     0.0256          0.0256         0
  setup_basicvariables           0.0056          0.0056         0
  setup_latt_hamil               0.0014          0.0014         0
  setup_init_data_node           0.0256          0.0256         0
  hel_setup_structfactors        0.0256          0.0256         0
  hlt_create_supercell           0.0014          0.0014         0

 Timing statistics (seconds)
  Function                       Wall   Max_Wall    Max_CPU  Node
  setup_main                     3.70       3.70       3.70     0
  setup_readinput                3.48       3.48       3.48     0
  montecarlo_main                0.21       0.21       0.21     0
  montecarlo_sweep               0.21       0.21       0.20     0
  io_readparameterfile           3.48       3.48       3.48     0
  setup_latt_hamil               0.22       0.22       0.22     0
  hlt_add_harmonic_didi          0.22       0.22       0.22     0

 Total memory usage:     0.03 MB
 Total WALL:           3.911 seconds
*****************************************************************
*                                                               *
*               Succesfully ending SCALE-UP                     *
*                                                               *
*****************************************************************
